src/transport/CASESession.h

```diff
@@ -155,6 +218,16 @@ class DLL_EXPORT CASESession
 
     Protocols::SecureChannel::MsgType mNextExpectedMsg = Protocols::SecureChannel::MsgType::CASE_SigmaErr;
 
+    Hash_SHA256_stream mCommissioningHash;
+    P256PublicKey mRemotePubKey;
+    P256Keypair mEphemeralKey;
+    P256Keypair mDeviceOpKeys;
+    const uint8_t * mDeviceOpCert;
```

Comment:
I think it'll be better to define a data structure (e.g. similar to `P256Keypair`) which can hold the op cert. Currently, it requires the caller of `WaitForSessionEstablishment` and `EstablishSession ` to hold on to the buffer that was passed in as parameter.